### DISCUSSION TITLE
Mute indicator

### DISCUSSION THREAD

**Discussion by rich-martin** [ttscoff/bunch-docs#276] [Q&A] [answered]
I was searching for an app to mute my mic at a system level with a keyboard shortcut for whichever meeting tool I'm in. I'm hoping to add a small indicator, trying to do it with this. The muting and notifications work, but the indicator doesn't. Any ideas?

```
---
shortcut: $m
---

* set volume input volume 0 
(notify Mic Muted)
(display ~/Users/Admin/Downloads/mute.png d:0 150x63 0,0 #clear a:50% c:n l:w)

! * set volume input volume 100 
!(notify Mic Active)
!(display ~/Users/Admin/Downloads/mic.png d:0 150x63 0,0 #clear a:50% c:n l:w)
```

**Comment by ttscoff** [comment, marked as answer]
I tested with my own images and your command works fine to put an image in the lower left corner of the main display. However, I think your paths are faulty, you don't need the `/Users/Admin` part, just `~/Downloads/mic.png`.

**Reply by rich-martin** [reply]
It does work it! But it puts the image below windows. Anyway to make it appear on top of all windows?

**Comment by ttscoff** [comment]
BTW, I've had good luck with an app called Mikrofon but its availability comes and goes, and it doesn't seem to be available on the US Mac App Store right now. My old favorite was [Shush](https://apps.apple.com/us/app/shush-microphone-manager/id496437906?mt=12), but I ran into some compatibility problems with my newer audio interfaces.

**Comment by ttscoff** [comment]
Change l:w (level:wallpaper) to l:f (level:float)

Thanks,
Brett

**Reply by ttscoff** [reply]
but note that it will function as a window now and you won't be able to click through it without moving it, and even without chrome or background, you may still get a titlebar when you hover over it. It looks like you copied the dimensions of my images, and you'll probably have better luck if you change the WxH part of the command to match the pixel dimensions of your mic/mute images.
